canonical-websites/www.ubuntu.com
# MAAS on product page template

## Done

 - Created product page template
 - Moved MAAS page to product page template
 - Created contact page for MAAS

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/server/maas](http://0.0.0.0:8001/server/maas)
- Check the page visually in all viewports - against the [copydoc](https://docs.google.com/document/d/1A6Cp2zpbk7I5Rl_Novs4awbR5L97Bf9_DwFyU1D76pA/edit#)
- Check the [contact page](http://0.0.0.0:8001/server/maas/contact-us) works correctly


## Issue / Card

Fixes #2001
